cccs-build/superset/requirements.txt

```diff
@@ -0,0 +1,10 @@
+# Required for OAuth
+authlib==0.15.4
```

Comment:
I see psycopg2==2.9.1 didn't make it over here. Did it turn out to not be needed?

Reply:
Yeah, the base image now includes `psycopg2-binary` by default and I was kinda hoping that would be sufficient to make it work. So far, I haven't seen anything broken because of this.